odbc/src/api/functions.rs

```diff
@@ -1024,6 +1024,12 @@ pub unsafe extern "C" fn SQLExecDirectW(
         || {
             let query = input_text_to_string_w(statement_text, text_length as usize);
             let mongo_handle = MongoHandleRef::from(statement_handle);
```

Comment:
Probably safe on debug.

Reply:
I moved several of the tracing that should really be debug to debug (for example, there is no reason to log handle freeing on info). We should include in the documentation that debug level will log queries which can potentially leak sensitive column and table names. When we include the parser in the driver later on we could do a ticket to obscure the query identifiers.

Reply:
I believe SQLFreeHandle used Info level for a specific reason. Can you double-check with @terakilobyte ?

Otherwise, yes, I am fine with the idea to log queries in debug and document that. We can look at obfuscating sensitive data later if needed. As far as I know Postgres, Mysql, etc... log their queries. And so does mongod, right?

Reply:
We have to log everything up through SQLDriverConnectW as Info because we can't know what their desired log level is, so the SQLFreeHandle call was left at Info to "complete" the log and not leave an impression that something was wrong from the user's perspective if they didn't see it in the log.